api/lambdas/sqs-to-converter/index.js

```diff
@@ -46,7 +47,8 @@ const fhirConverter = axios.create({
     clarifyTimeoutError: true,
   },
 });
-
+const OSSApi = axios.create();
```

Comment:
It'd be good to keep using first letters lower case for instances and upper to types. :)